### DISCUSSION TITLE
302 Moved Temporarily with Node.js Micros

### DISCUSSION THREAD

**Discussion by shokhboz-abdullaev** [deta/space-docs#111] [Q&A]
Hello everyone! Deta Space is such an awesome platform and I have really been loving it and started using it literally today! I tried to deploy my Node backend application (Express) and it has a few routes for my front-end website which is currently deployed to Netlify. But when I open my front-end site the content that should be coming from backend is not displayed and when I opened network tab while inspecting, I discovered that the backend url is correct but the response from it is temporarily moved. How can I fix it? 

Thanks in advance for your kind help!

**Comment by lihugang** [comment]
Set `public: true` in your Spacefile?

**Reply by lihugang** [reply]
https://github.com/orgs/deta/discussions/107
